host/host.go

```diff
@@ -170,6 +171,17 @@ func runDaemon(args *docopt.Args) {
 	var backend Backend
 	var err error
 
+	// use a zpool backing file size of either 70% of the root device size,
+	// or 100GB if that can't be determined.
+	var zpoolBackingFileSize int64
+	var rootDev syscall.Statfs_t
+	if err := syscall.Statfs("/", &rootDev); err == nil {
```

Comment:
Can we point this at the directory where the sparse file will live? It might not be on the root device.